### ISSUE TITLE
Multi-task training

### ISSUE THREAD

**Issue by faranaksh** [vlfeat/matconvnet#1146]
Dear all,

I am training a dagg network with two objective functions, i.e. a multi-tasking network. I have some queries in this regard and I really appreciate your help.

1. If we notate loss 1 with _L1_ and loss 2 with _L2_, is the network considering the total loss as _L1_+_L2_?
2. Should the two loss functions reside in a similar range? One of the loss functions yields values around 100 and the other values less than 5. For example, if _L1_ = 120, _L2_ is 1.5. Do I have to take them in a similar range?
3. If I want to give more importance to one objective function, is there a way to make the total loss function something like _a_ * _L1_ + _b_ * _L2_? Where I can make such modification?

Thanks

**Comment by haoliyoupai09** [comment]
I would like to know, have you solved your problem now? Could you share me the solution?
Thank you very much.
